lib/molecule-dev-environment.js

```diff
@@ -69,7 +151,28 @@ export default {
       "Shell.js",
       "Testcafe.js",
       "Cpp.js",
-    ].forEach(tool => provideDevtool(store, DevToolsControllerInstance)(tool));
+    ].map(plugin => {
```

Comment:
A few suggestions:

* Use `const` rather than `let` where applicable.

* Using `map` suggests that you're going to return an object, eg `let doubles = myArray.map(x => x * 2)`; you shouldn't use `array.push` inside map. I recommend replacing `pluginsTable.push` with `provideDevtool`, removing `pluginsTable` and replacing `.map` with `.forEach`.

* Calling your variable `pluginName` suggests it will store the name of your plugin, not an array of strings that include the name; I recommend writing `const pluginName = plugin.split(/\/|\./)[0]` instead;

* The `if (pluginsTable !== [])` is extraneous; foreach can handle an empty array.